homeassistant/components/rachio/device.py

```diff
@@ -78,11 +79,18 @@ def setup(self, hass):
             # webhooks are normally a list, however if there is an error
             # rachio hands us back a dict
             if isinstance(webhooks, dict):
-                _LOGGER.error(
-                    "Failed to add rachio controller '%s' because of an error: %s",
-                    controller[KEY_NAME],
-                    webhooks.get("error", "Unknown Error"),
-                )
+                if webhooks.get("code") == PERMISSION_ERROR:
+                    _LOGGER.info(
+                        "Not adding controller '%s', only controllers owned by '%s' may be added.",
```

Comment:
We don't end logging messages with period.
```suggestion
                        "Not adding controller '%s', only controllers owned by '%s' may be added",
```

Reply:
Sorry, on a roll with these periods today I guess!